pavelib/servers.py

```diff
@@ -240,7 +240,7 @@ def update_db():
     """
     settings = getattr(options, 'settings', DEFAULT_SETTINGS)
     for system in ('lms', 'cms'):
-        sh(django_cmd(system, settings, 'migrate', '--traceback', '--pythonpath=.'))
+        sh(django_cmd(system, settings, 'migrate', '--fake-initial', '--traceback', '--pythonpath=.'))
```

Comment:
Now we're thinking it might be too much to always use ``--fake-initial``, and it should be an option to the paver command.